EventFilter/SiStripRawToDigi/src/SiStripFEDBufferComponents.cc

```diff
@@ -845,6 +845,7 @@ namespace sistrip {
       case READOUT_MODE_SCOPE:
         //scope mode is always real
         setReadoutModeBits(readoutMode);
+        [[fallthrough]];
```

Comment:
It looks like there is some odd logic behind...

As it is now, all case's but the default one foresee
```C++
    setReadoutModeBits(readoutMode);
```
and therefore one could simplier remove L846, L847, L848, L862, L863 (and at this point even L865).

Could you please confirm that this is indeed the intended behaviour @OlivierBondu @alesaggio ?

Reply:
@pieterdavid FYI

@perrotta, Olivier has left CMS, please update the list of reco contact for strips accordingly.

Reply:
> @pieterdavid FYI
> 
> @perrotta, Olivier has left CMS, please update the list of reco contact for strips accordingly.

Done in https://twiki.cern.ch/twiki/bin/view/CMS/RecoContacts

Reply:
This simplification should be correct and safe indeed.